### DISCUSSION TITLE
ödev teslim ederken git hatası

### DISCUSSION THREAD

**Discussion by yunusemreerkesikbas** [19/classroom#38] [C]
hocam ödevi teslim ederken git hatası veriyor user.name ve user.email configure edilmedi diyor ancak sizin dediğiniz üzere global user.name ve global user.email tanımlamaları da yaptım githubdaki kullanıcı adıma göre name ve mail girdim ancak gene olmadı

**Comment by yunusemreerkesikbas** [comment]
![gitoutpuyt](https://user-images.githubusercontent.com/72819723/159000718-d0472249-b5a5-474b-9a2f-34e149c76eb1.png)
![komut](https://user-images.githubusercontent.com/72819723/159000755-b4c66ebc-5248-4a4a-945b-08dde4ef5957.png)

**Comment by gulcanyildiz** [comment]
Terminale aşağıdaki gibi yazman gerekiyor.

```sh
git config --global user.email "20060364@stu.omu.edu.tr"
git config --global user.name "yunussemreerkeskbs"
```

**Reply by yunusemreerkesikbas** [reply]
dediğiniz şekilde de yaptığımda aynı hatayı alıyorum acaba name ve email isimlendirmesini mi yanlış yapıyorum yani geçerli olması için hangi mail ve name kulanmam lazım

**Reply by yunusemreerkesikbas** [reply]
email ve name bölümünü configure ettiğinizden emin olun diyor ama komut satırında user.name ve user.email kısmını sorguladığımda veriyor  bilgilerimi

**Reply by gulcanyildiz** [reply]
VS Code terminalinde tekrar dene. Olmazsa hata kodlarını gönderebilirsin. Ödevi acil teslim etmek istiyorsan tarayıcı üzerinden VS Code kullanarak komit edebilirsin.

**Reply by yunusemreerkesikbas** [reply]
tekrar denedim ama olmadı ben tarayıcı üzerinden teslim edeyim
